lib/transatl.g

```diff
@@ -169,9 +169,8 @@ BindGlobal("CheckSynonyms", function(arg)
   for p in pairs do 
     Print("Checking pair ", p, "\n");
     for n in allnames do
-      pos := PositionSublist(n, p[1]);
-      if pos <> fail then
-        n2 := SubstitutionSublist(n, p[1], p[2]);
+      n2 := ReplacedString(n, p[1], p[2]);
+      if not IsIdenticalObj(n) <> IsIdenticalObj(n2) then
```

Comment:
??? (It does not help to run tests which do not call the changed function.)
Shouldn't this be:
```
    if not IsIdenticalObj(n, n2) then
```